gamestonk_terminal/stocks/fundamental_analysis/dcf_view.py

```diff
@@ -77,8 +77,11 @@ def create_workbook(self):
             self.run_audit()
 
         trypath = os.path.join(
-            "..",
-            "GamestonkTerminal",
+            os.path.abspath(
+                os.path.join(
+                    os.path.dirname(os.path.dirname(os.path.dirname(__file__))), ".."
+                )
+            ),
```

Comment:
This seems to do the trick, works in both cases when you open terminal.py from the main folder or from somewhere else.

Reply:
Look into my other comment for a more elegant solution